frank-doc-doclet/src/main/java/org/frankframework/frankdoc/wrapper/FrankAnnotationDoclet.java

```diff
@@ -42,34 +65,34 @@ public boolean isPublic() {
 
 	@Override
 	public Object getValue() throws FrankDocException {
-		List<Object> candidates = getField("value");
-		return getValueFromFieldRemovingRepetition(candidates);
+		return getValueOf("value");
 	}
 
 	@Override
 	public Object getValueOf(String fieldName) throws FrankDocException {
-		List<Object> candidates = getField(fieldName);
-		return getValueFromFieldRemovingRepetition(candidates);
-	}
-
-	private List<Object> getField(String fieldName) {
-		List<Object> candidates = Arrays.asList(annotation.elementValues()).stream()
+		Optional<Object> candidate = Arrays.asList(annotation.elementValues()).stream()
 				.filter(ev -> ev.element().name().equals(fieldName))
 				.map(ev -> ev.value().value())
-				.collect(Collectors.toList());
-		return candidates;
+				.findAny();
+		if(candidate.isPresent()) {
+			return parseAnnotationValue(candidate.get());
+		} else {
+			return getDefaultValue(fieldName);			
+		}
 	}
 
-	private Object getValueFromFieldRemovingRepetition(List<Object> candidates) throws FrankDocException {
-		if(candidates.isEmpty()) {
-			return null;
+	private Object parseAnnotationValue(Object raw) throws FrankDocException {
+		// Annotations with an enum-typed field are not supported. To do that, we would
+		// parse the @EnumLabel annotation, but doing so is the job of the "model" package.
+		// Implementing enum-typed annotation fields generally in the wrapper API would
+		// confuse the tasks of the wrapper API and the model, making the code
+		// much more complicated.
```

Comment:
Geldt dit commentaar nog steeds?

Reply:
Nee, dit comment klopte niet meer. Het is nu weg.